lib/stripe.js

```diff
@@ -83,6 +83,9 @@ var resources = {
     Transactions: require('./resources/Issuing/Transactions'),
     Disputes: require('./resources/Issuing/Disputes'),
   }),
+  Sigma: resourceNamespace('sigma', {
+    ScheduledQueryRuns: require('./resources/Sigma/ScheduledQueryRun'),
```

Comment:
It's kind of annoying, but I think the convention in Node is to have the filenames plural. Can we make this `ScheduledQueryRuns`?

Reply:
ah nice catch on that one. I just fixed it if you can take a second look